test/cppsuite/test_harness/workload/random_generator.cxx

```diff
@@ -37,25 +38,28 @@ random_generator::instance()
 }
 
 std::string
-random_generator::generate_string(std::size_t length)
+random_generator::generate_string(std::size_t length, characters_type type)
```

Comment:
In terms of class design, it would be better to make `random_generator` an abstract class. Add two random generators, one for alphabet another one for alphanum. They both would be a subclass of `random_generator`.
Or at least init random_generator with characters_type rather than passing it into methods.